docs/source/staking/stake_initialization.rst

```diff
@@ -228,7 +228,7 @@ Once you have created one or more stakes, you can view all active stakes for con
     ╒═══════╤══════════╤═════════════╤═════════════╤═══════════════╕
     │   Idx │ Value    │   Remaining │ Enactment   │ Termination   │
     ╞═══════╪══════════╪═════════════╪═════════════╪═══════════════╡
-    │ 	0   │ 30000 NU │      	  31 │ Jun 19 2020 │ Jul 19 2020   │
+    │ 	0   │ 30000 NU │      	  5  │ Jun 19 2020 │ Jul 17 2020   │
```

Comment:
This is a bit confusing. I guess until there's < 4 weeks to go, it will still print 5 periods remaining. Perhaps to show this better we could set the illustrative Termination date as July 23rd (i.e. 4 weeks and 6 days)

Reply:
Of note is that I manually edited these outputs.

The remaining periods is calculated as follows:

```python
    @property
    def periods_remaining(self) -> int:
        """Returns the number of periods remaining in the stake from now."""
        current_period = self.staking_agent.get_current_period()
        return self.final_locked_period - current_period + 1
```

Technically the value could be anything <= 5. However, since this is a new stake, the current date is Jun 19th, 2020. You'll notice that the old value was for 30 periods (30 days), but remaining was 31 because of the same calculation;  because the first commitment is for the next period. Hoping that @vzotova can confirm whether this edit is correct...

I can modify the termination date if that is too confusing, but it was essentially the same thing we had before.

Reply:
looks correct, ideally to try it on lynx

Reply:
Added CLI output from lynx operations.